### PR TITLE
Add locks to starboard and include example of level-only starring

### PR DESCRIPTION
**Pull request by DarkView**

### CODE REVIEW

File: backend/src/plugins/Starboard/StarboardPlugin.ts [hunk 1]
```diff
@@ -88,6 +88,27 @@ export const StarboardPlugin = zeppelinGuildPlugin<StarboardPluginType>()("starb
                 basic:
                   enabled: true
       ~~~
+
+      ### Limit starboard to a specific level (and above)
+      This is identical to the channel-specific starboard above, but only works for a specific level (>=50).
+      It is noteworthy that this only properly works if the starboard requires just one star.
```
Comment (Dragory) [inline backend/src/plugins/Starboard/StarboardPlugin.ts:94]: Why is this?

Comment (DarkView) [reply backend/src/plugins/Starboard/StarboardPlugin.ts:94]: Due to a request from a bot users and a few others saying they wanted a level-specific starboard as well
https://canary.discord.com/channels/473085256233123841/739034889399042129/779047601454579744

Comment (Dragory) [reply backend/src/plugins/Starboard/StarboardPlugin.ts:94]: Oh, sorry, I meant this line specifically:
> It is noteworthy that this only properly works if the starboard requires just one star
Doesn't it work with multiple stars too? Just that it doesn't count the stars from those below the specified level.

Comment (DarkView) [reply backend/src/plugins/Starboard/StarboardPlugin.ts:94]: Oh, my bad i misunderstood when the reactions get counted.

**Pull request merged**